nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [Optional] Let's modify the error raising like this:

  `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`